between random: 245180626
between random: 245180622
between random: 245180626
between random: 9034
between random: [132,171]
between random: [132,171]
between random: [132,171]
between random: [132,171]
between random: [132,171]
between random: [132,171]
between random: [132,171]
between random: [132,171]
between random: [132,171]
between random: [132,171]
between random: [132,171]
between random: [132,171]
between random: [132,171]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [13,209]
between random: [111,29]
between random: [111,29]
between random: [66,42]
between random: [66,42]
between random: [202,142]
between random: [202,142]
between random: [202,142]
"    require ~sent_pkt(scid,pkt.ptype,pkt.seq_num);  # [4]
"
/usr/local/lib/python2.7/dist-packages/ivy/include/1.7/quic_packet.ivy: line 397: error: assumption failed
output directory: temp/479
implementation directory: /home/chris/TVOQE_UPGRADE_27/quic/quic-go/server/
implementation command: ./server -p 4443 127.0.0.1
../quic_server_test_new_token_error (0) ...
implementation command: ['./server', '-p', '4443', '127.0.0.1']
quic_process pid: 8706
timeout 100 ./build/quic_server_test_new_token_error seed=768 the_cid=0 server_cid=1 client_port=4987 client_port_alt=4988
client return code: 1
FAIL
error: 1 tests(s) failed
